docs/source/tutorials/cmip6-ecs.ipynb

```diff
@@ -0,0 +1,3690 @@
+{
```

Comment:
<p>Could we add docstrings to these tasks? They don't need to be elaborate. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds-funnel/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>